### ISSUE TITLE
Leak of memory

### ISSUE THREAD

**Issue by cometkim** [tamayika/gaq#3]
First of all, thank you for your great work!

I recently updated CLI version 0.5 to 0.9 using `go get -u github.com/tamayika/gaq`
It's no longer working and increases steadily until memory usage reaches 100%.

![image](https://user-images.githubusercontent.com/9696352/48855967-8e4ec780-edf8-11e8-8d0d-5db3dad9c74f.png)

(killed the process at end of graph)

**Comment by tamayika** [comment]
Thank you for the report.
Can you provide the minimal repro with query and input?

**Comment by cometkim** [comment]
`cat ./[ANY_FILES] > gaq "TypeSpec"` in [here](https://github.com/mattermost/mattermost-server/tree/master/model)

OS: Ubuntu Xenial 
Go: v1.11.2

**Comment by tamayika** [comment]
It's participle bug.
Please try `go get github.com/alecthomas/participle@28bd879b3ab5` with go mod enabled.

**Comment by cometkim** [comment]
```
➜  gaq git:(master) GO111MODULE=on go get github.com/alecthomas/participle@28bd879b3ab5
go: finding github.com/spf13/cobra v0.0.3
go: finding github.com/inconshreveable/mousetrap v1.0.0
go: finding github.com/spf13/pflag v1.0.3
go: finding golang.org/x/exp v0.0.0-20181112044915-a3060d491354
go: finding github.com/alecthomas/participle 28bd879b3ab5
➜  gaq git:(master) GO111MODULE=on go install
go: downloading github.com/spf13/cobra v0.0.3
go: downloading github.com/spf13/pflag v1.0.3
go: finding github.com/alecthomas/participle v0.1.1-0.20181115125005-28bd879b3ab5
```

It works well now. Thank you for investigation.

Again, this and vscode-go-ast-explrorer is amazing!

**Comment by tamayika** [comment]
Thank you for the report.
If you have any other problems, please ask me anytime!

**Comment by tamayika** [comment]
@cometkim 

I updated gaq to support particple 0.2.0 and released 0.0.12.
I also added binary releases built with go mod at [Releases](https://github.com/tamayika/gaq/releases), so if you don't want to build, please use these binaries.

@alecthomas 

very very thanks again!
